styles.css

```diff
@@ -0,0 +1,89 @@
+html {
+  font-size: 100%;
+  line-height: 1.5rem;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0;
+  min-height: 100vh;
+  max-width: 1400px;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+  display: block;
+}
+
+/* toolbar */
+.header-nav {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  padding: 25px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 58px;
+  align-items: center;
+  padding: 64px 24px;
+  position: absolute;
+  background: url(images/header-illsutration-mobile@2x.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] If you want to create a CSS rule for multiple elements, use a class attribute instead. For example, instead of using `display: flex;` multiple times, you can create a class
``` css
  .flex {
    display: flex;
  }
```
and add it to all elements that need the display to be flex.